z3c/dependencychecker/package.py

```diff
@@ -176,3 +177,21 @@ def _get_sources_top_level(self):
             os.sep,
         )
         sys.exit(1)
+
+
+class Package(object):
```

Comment:
Can you add a short docstring? I think you mean "something like zest.releaser or z3c.dependencychecker" with "package". But better make it explicit.

Reply:
Sure, no problem